app/networkapi/people/models.py

```diff
@@ -97,6 +97,12 @@ class Person(SortableMixin):
         null=True,
         blank=True,
     )
+    interview_url = models.URLField(
+        max_length=500,
```

Comment:
are you sure about this? it seems a weird limit for a thing that is by spec defined as 4096 characters max